Core/src/main/java/org/tribuo/sequence/SequenceDataset.java

```diff
@@ -211,5 +213,24 @@ public FlatDataset(SequenceDataset<T> sequenceDataset) {
         }
     }
 
+    protected static List<SequenceExample<?>> deserializeExamples(java.util.List<org.tribuo.protos.core.SequenceExampleProto> examplesList, Class<?> outputClass, FeatureMap fmap) {
```

Comment:
Can you remove the fully qualified types & rename this to `deserializedSequenceExamples`?